src/main/java/frc/robot/utilities/FieldPoseUtil.java

```diff
@@ -17,6 +17,8 @@
 import frc.robot.Constants.FieldConstants;
 
 public class FieldPoseUtil {
+  private ReefPose m_reefPose = ReefPose.TWO;
```

Comment:
`m_reefPose` never changes, so I don't see how `whichAlgaeHeight()` can be always correct. It feels like `whichAlgaeHeight()` should be taking a parameter.